backend/app/services/adapters/task_restore.py

```
@@ -0,0 +1,199 @@
+logger = logging.getLogger(__name__)
+
+
+class TaskRestoreService:
+    """Service for restoring expired tasks."""
+
+    def restore_task(
+        self,
+        db: Session,
+        task_id: int,
+        user: User,
+        message: Optional[str] = None,
```

Ruff (0.14.14)

[warning] Unused method argument: `message`

(ARG002)